modules/common/src/main/PackageObject.scala

```diff
@@ -190,8 +190,11 @@ trait WithPlay { self: PackageObject =>
 
   implicit final class LilaPimpedFiniteDuration(self: FiniteDuration) {
 
-    def toTenths: Long = self.toMillis / 100
+    def roundSeconds = math.round(self.toMillis.toDouble / 1000)
+    def roundTenths = math.round(self.toMillis.toDouble / 100)
```

Comment:
`50 / 100` rounds down to `0` ≠ `1.0 = math.round(50.toDouble / 100) = math.round(0.5)`